recipes/Drop-seq_tools/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "Drop-seq_tools" %}
+{% set version = "1.13" %}
+
+about:
+  home: http://mccarrolllab.com/dropseq/
+  license: MIT License
+  license_family: MIT
+  summary: |
+    Package fot the analysis of Drop-seq data developed by Jim Nemesh in the McCarroll Lab
+package:
+  name: dropseq_tools
+  version: {{ version }}
+
+build:
+  number: 1
+
+source:
+  url: http://mccarrolllab.com/download/1276/{{ name }}-{{ version }}.zip
+  md5: 28533edd82867ed75867b05785cf8834
+
+requirements:
+  run:
+    - openjdk >=7
+
+test:
+  commands:
+    - '[[ $(TrimStartingSequence -- --version 2>&1) == "1.13(7bed8f4_1513008033)" ]]'
```

Comment:
Include a few more tests e.g. the --help from the nodes?

Reply:
ok can do this, give a sec